### ISSUE TITLE
Help me understand MVC 6 support for dotnet5.6 vs dotnet5.5

### ISSUE THREAD

**Issue by nathanielcook** [aspnet/Mvc#4060]
Looking at the packages folder, `Microsoft.AspNet.Mvc 6.0.0-rc2-16944` supports only TFMs `dotnet5.6` and `net451`, whereas `6.0.0-rc2-16928` supports TFMs `dotnet5.5` and `net451`. I am currently using the `1.0.0-rc2-16357` coreclr runtime (x64) on OSX, which doesn't seem to recognize `dotnet5.6`. Is there any reason that `Microsoft.AspNet.Mvc 6.0.0-rc2-16944` does not include support for both `dotnet5.5` and `dotnet5.6`? Is it to force an upgrade of the runtime? I apologize if my question reflects a misunderstanding of terminology. I am trying to understand but I could be getting it wrong.

The main reason I am asking about this is that I am able to restore all of my project.json dependencies using the `aspnetvnext` myget feed with the version of my packages in project.json set to `rc2-*`. But with the `aspnetcidev` myget feed, the `"Microsoft.AspNet.Mvc": "6.0.0-rc2-*"` dependency is not found. It seems to be because the latest rc2 package has a dotnet5.6 folder and no dotnet5.5 folder.

FYI I'm not sure if it makes a difference, but I'm still using `dnu restore`; I'm not using the dotnet cli yet. I have read of some problems with the dotnet cli and was waiting for it to stabilize a bit more. Would that have anything to do with it by chance?

**Comment by pranavkm** [comment]
We had to switch to `dotnet5.6` because one of our dependencies (`System.Runtime.Loader`) was retargeted to `dotnet5.6` sometime during RC2. 
Additionally, we recently renamed all our packages (and the binaries, namespaces in our libraries) to AspNetCore (See https://github.com/aspnet/Announcements/issues/144). The package on `aspnetcidev` is now named `Microsoft.AspNetCore.Mvc`. `dnu restore` should work for now although pointing to `aspnetcidev` might result in transition pains (https://github.com/aspnet/Announcements/issues/150).
